meson.build

```diff
@@ -156,7 +161,19 @@ endif
 configuration.set_quoted('CFGOPTDEF_CONFIG_PATH', get_option('configdir'), description: 'Configuration path')
 
 # Set FN_NO_RETURN macro
-configuration.set('FN_NO_RETURN', '__attribute__((__noreturn__))', description: 'Indicate that a function does not return')
+if cc.has_function_attribute('__noreturn__')
```

Comment:
@dwsteele: 
I wrapped ALL my changes inside if-s that check for msvc, except for this one.

However, it seems that my changes broke ALL platforms.
Any ideas?

Reply:
This should work bd61e9d5ef. See the reference linked in that commit for valid values.

I also merged from main to cleanup cruft in the branch.